substrate/client/executor/polkavm/src/lib.rs

```diff
@@ -43,13 +44,6 @@ impl WasmInstance for Instance {
 		name: &str,
 		raw_data: &[u8],
 	) -> (Result<Vec<u8>, Error>, Option<AllocationStats>) {
-		let Some(method_index) = self.0.module().lookup_export(name) else {
-			return (
-				Err(format!("cannot call into the runtime: export not found: '{name}'").into()),
-				None,
-			);
-		};
```

Comment:
Still missing this piece of code.

Reply:
OK so then probably something else must be used than `call_typed`. Can you give advice on that?

Reply:
Other issues resolved except this as I'm not sure how to sort this out. Should `prepare_call_untyped()` be used?

Reply:
@koute Based on our discussion I did `git revert` for my earlier fixup and returned this, and then use program counter to address the entry point.